File: programmers/예상 대진표.py
def solution(n, a, b):
    answer = 0
    # 토너먼트이므로 a와 b는 반드시 만난다는 것을 이용 -> n 파라미터는 필요가 없음
    while True:
        answer += 1
        a = int(a / 2 + 0.5)
        b = int(b / 2 + 0.5)
        
        if a == b:
            break
    return answer

File: programmers_level2/예상_대진표.cpp
#include <iostream>

using namespace std;

int solution(int n, int a, int b)
{
    int answer = 0;
    while (a != b) {
        a = (a / 2) + (a % 2); // 홀수일 경우 +1
        b = (b / 2) + (b % 2);
        ++answer; // 한 라운드의 끝
    }

    return answer;
}